packages/ui-kit/src/components/lord-icon-client-side.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useRef, useState } from 'react';
+import Lottie, { LottieRefCurrentProps } from 'lottie-react';
+
+type LordIconProps = {
+  iconName: string;
+  loop: boolean;
+  autoplay: boolean;
+  style: object;
+};
+
+const defaultProps = {
+  loop: false,
+  autoplay: false,
+};
+
+const LordIcon = (props: LordIconProps) => {
+  const [animationData, setAnimationData] = useState();
+  const lottieRef = useRef<LottieRefCurrentProps>(null);
+  const { iconName, ...rest } = props;
+
+  useEffect(() => {
+    const loadAnimationData = async () => {
+      try {
+        const data = await import(
+          `@commercetools-docs/ui-kit/dist/icons/lord-icon/${iconName}.json`
+        );
+        setAnimationData(data.default);
+      } catch (error) {
+        console.error(`Error loading icon animation: ${error}`);
+      }
+    };
+    loadAnimationData();
+  }, [iconName]);
+
+  const handleMouseEnter = () => {
+    lottieRef.current?.play();
+  };
+
+  const handleMouseLeave = () => {
```

Comment:
TBH I'm not conviced the animation should abruptly stop when leaving the mouse.  Normally they continue the initialized loop.  
The lordicon custom element has the triggers preconfigured, there's "hover",  "loop-on-hover", "loop", "morph" etc.  ( https://github.com/tomwilusz/lord-icon-element/blob/master/examples/triggers.html ) 

Their "hover" trigger only acts on mouse enter ( https://github.com/tomwilusz/lord-icon-element/blob/master/src/triggers/hover.ts ) 

Interaction with content and design people could also be easier if we use the lordicon custom element because the exact behavior of the triggers it provides are built into the lordicons website and they can try out there.  ( e.g. click "HTML" on one of the icons here: https://lordicon.com/icons/wired/outline )